src/Trie.js

```diff
@@ -7,33 +7,20 @@ const getClosestNode = require('./getClosestNode')
 /**
   Trie class
   @name Trie
-  @example const store = new Trie()
+  @example const store = new Trie(...args)
 */
-class Trie {
+class Trie extends Map {
   rootSocket: Object
-  cache: Map<string, any>
-  size: number
 
-  constructor () {
+  constructor (args?: any) {
+    super(args)
+    const self = this
     this.rootSocket = {}
-    this.cache = new Map()
-    this.size = 0 // TODO
-  }
-
-  entries () {
-    // TODO
-  }
-
-  has () {
-    // TODO
-  }
-
-  keys () {
-    // TODO
-  }
-
-  values () {
-    // TODO
+    if (args) {
+      args.forEach((value, key, table) => {
```

Comment:
We don't use `table` anywhere so why define it?

Reply:
My bad. We don't need it here